Fixes for 1.3.5

mocchira opened the pull request:
Two fixes there.
- docs: Add the FAQ about timeout during PUT
  - Preview can be found here: https://mocchira.github.io/leofs/faq/administration/#what-should-i-do-when-a-timeout-error-happen-during-upload-a-very-large-file
- gateway: Fixes for #770
- storage: Fixes for #769

yosukehara commented on docs/faq/administration.md:
````diff
@@ -106,3 +106,28 @@ If you changed a WRONG node name before stopping the daemon, As a result, when a
+# EXPECTED_BANDWIDTH_IN_BYTES = 1048576 / (1000 / 1000) * 128 = 134217728(128MB)
+```
+
+is considered as a valid configuration.
````
It is able to easily read to connect this line with the above sentence as below:


For example, say we have 1Gbps(128MB/sec) stable bandwidth and 128 concurrent uploads at most then we could be ready to 1MB per connection so that is considered as a valid configuration.

mocchira commented:
@yosukehara PTAL
- Fixed what you pointed out
- Added the sentence about what is the difference between decreasing the buffer size and increasing the timeout

yosukehara commented:
@mocchira I've reviewed your PR, LGTM. Merged.
